recipes-core/meta/meta-toolchain.bbappend

```diff
@@ -2,3 +2,10 @@
 # /usr/local/oe-sdk-hardcoded-buildpath, so avoid causing errors for
 # outgoing symlinks that are there by design.
 CHECK_SDK_SYSROOTS = "0"
+
+# Add the package sources needed when building the SDK with BUILD_IMAGES_FROM_FEEDS
+IPK_FEED_URIS:append:task-populate-sdk = "${@oe.utils.conditional('BUILD_IMAGES_FROM_FEEDS', '1', '\
+    NIOE-dummy-sdk##${NILRT_LOCAL_FEED_URI}/sdk-provides-dummy-target \
+    NIOE-nativesdk##${NILRT_LOCAL_FEED_URI}/${SDK_ARCH}-${SDKPKGSUFFIX} \
+    NIOE-dummy-nativesdk##${NILRT_LOCAL_FEED_URI}/sdk-provides-dummy-${SDKPKGSUFFIX} \
+', '', d)}"
```

Comment:
This won't immediately impact our toolchain builds as we disable `BUILD_IMAGES_FROM_FEEDS` when building currently. That being said, this would allow us to eventually back that out of openembedded-core.

Reply:
Back what out of OE-core? BIFF or logic to disable BIFF when building the toolchain targets?

Reply:
The logic to disable BIFF when building the toolchain targets. We have two custom changes in OE-core that I don't think were upstreamed to do that ([here ](https://github.com/ni/openembedded-core/commit/ddec9e153978b261f350cbb56b662abbbd5ca378) and [here](https://github.com/ni/openembedded-core/commit/312af55e6cd0b3f57b3e66f57bbd597aea564372)).